src/DocumentFormat.OpenXml/Packaging/OpenXmlPart.cs

```diff
@@ -380,15 +348,13 @@ public void FeedData(Stream sourceStream)
                 throw new ArgumentNullException(nameof(sourceStream));
             }
 
-            using (Stream targetStream = GetStream(FileMode.Create))
-            {
-                sourceStream.CopyTo(targetStream);
-            }
+            using Stream targetStream = GetStream(FileMode.Create);
+            sourceStream.CopyTo(targetStream);
         }
 
-        #endregion
+#endregion
```

Comment:
can you revert the spacing changes here?

Reply:
Done. I've done that for all `#region` and `#endregion` occurrences. I don't remember changing those.